MATLAB/Utilities/filtering.m

```diff
@@ -22,7 +22,7 @@
 
 if parker
 	proj = permute(ParkerWeight(permute(proj,[2 1 3]),geo,angles,parker),[2 1 3]);
-	diff_angles = diff(angles); angle_step = mean(abs(diff_angles)); % to be used later
+	diff_angles = diff(angles(1,:)); angle_step = mean(abs(diff_angles)); % to be used later

```

Comment:
For SDK: we wouldn't really need this fix, because we compute `diff_angles` differently (angular difference between two 3D positions, since we don't store the angle) and we won't be affected by "jumps" between 360 and 0. However, it means we'd currently use all frames of a helical scan to compute `angle_step`, while TIGRE would only use the first revolution?

Reply:
When I said helical, I was lying. I meant scans where the source moves on a sphere. But then we wouldn't use FDK anyway, so I don't really know what this change is for. In any case, it doesn't affect us.